setup.py

```diff
@@ -18,59 +18,66 @@
-    description='Google Fonts Tools is a set of command-line tools'
-                ' for testing font projects',
+    url="https://github.com/googlefonts/tools/",
+    description="Google Fonts Tools is a set of command-line tools"
+    " for testing font projects",
     long_description=long_description,
-    long_description_content_type='text/markdown',  # This is important!
-    author=('Google Fonts Tools Authors: '
-            'Dave Crossland, '
-            'Felipe Sanches, '
-            'Lasse Fister, '
-            'Marc Foley, '
-            'Eli Heuer, '
-            'Roderick Sheeter'),
-    author_email='dave@lab6.com',
-    package_dir={'': 'Lib'},
-    packages=['gftools',
-              'gftools.actions',
-              'gftools.util',
-              'gftools.builder'],
-    package_data={'gftools.util': ["GlyphsInfo/*.xml", "UnicodeSections/*.json"],
-                  'gftools': [
-                      'template.upstream.yaml',
-                      "udhr_all.txt",
-                      "templates/*.html",
-                      "push-templates/*.html"
-                  ]
-                 },
+    long_description_content_type="text/markdown",  # This is important!
+    author=(
+        "Google Fonts Tools Authors: "
+        "Dave Crossland, "
+        "Felipe Sanches, "
+        "Lasse Fister, "
+        "Marc Foley, "
+        "Eli Heuer, "
+        "Roderick Sheeter"
+    ),
+    author_email="dave@lab6.com",
+    package_dir={"": "Lib"},
+    packages=["gftools", "gftools.actions", "gftools.util", "gftools.builder"],
+    package_data={
+        "gftools.util": ["GlyphsInfo/*.xml", "UnicodeSections/*.json"],
+        "gftools": [
+            "template.upstream.yaml",
+            "udhr_all.txt",
+            "templates/*.html",
+            "push-templates/*.html",
+        ],
+    },
     scripts=gftools_scripts(),
     zip_safe=False,
     classifiers=[
-        'Environment :: Console',
-        'Intended Audience :: Developers',
-        'Topic :: Text Processing :: Fonts',
-        'License :: OSI Approved :: Apache Software License',
-        'Operating System :: OS Independent',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 3'
+        "Environment :: Console",
+        "Intended Audience :: Developers",
+        "Topic :: Text Processing :: Fonts",
+        "License :: OSI Approved :: Apache Software License",
+        "Operating System :: OS Independent",
+        "Programming Language :: Python :: 2",
+        "Programming Language :: Python :: 3",
     ],
     python_requires=">=3.7",
-    setup_requires=['setuptools_scm>=4,<6.1'],
+    setup_requires=["setuptools_scm>=4,<6.1"],
     # Dependencies needed for gftools qa.
-    extras_require={"qa": ['fontbakery', 'fontdiffenator', 'gfdiffbrowsers']},
+    extras_require={
+        "qa": ["fontbakery", "fontdiffenator", "gfdiffbrowsers"],
+        "ninja": ["ninja"],
```

Comment:
I'm tempted to use ninja in `gftools qa` so I'm tempted to just install this dependency by default. wdyt?

Reply:
I think I'd like to have it as an optional extra for now until we've proved it a bit more.